.editorconfig

```diff
@@ -0,0 +1,21 @@
+; indicate this is the root of the project
+root = true
+
+[*]
+charset = utf-8
+
+end_of_line = LF
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+indent_style = space
+indent_size = 4
+
+[Makefile]
+indent_style = tab
+
+[makefile]
+indent_style = tab
+
+[*.go]
+indent_style = tab
```

Comment:
@Sergey1011010 I just notice that it is already using tabs to `.go` files. :)

Reply:
Yeah, I realized there was a section for *.go and deleted the comment :D